ui/components/app/add-network/add-network.js

```diff
@@ -0,0 +1,134 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { I18nContext } from '../../../contexts/i18n';
+import ActionableMessage from '../../ui/actionable-message';
+import Box from '../../ui/box';
+import Typography from '../../ui/typography';
+import {
+  ALIGN_ITEMS,
+  BLOCK_SIZES,
+  COLORS,
+  DISPLAY,
+  FLEX_DIRECTION,
+  TYPOGRAPHY,
+} from '../../../helpers/constants/design-system';
+import Button from '../../ui/button';
+
+const AddNetwork = ({
+  onBackClick,
+  onAddNetworkClick,
+  onAddNetworkManuallyClick,
+  featuredRPCS,
+}) => {
+  const t = useContext(I18nContext);
+
+  const nets = featuredRPCS
+    .sort((a, b) => (a.ticker > b.ticker ? 1 : -1))
+    .slice(0, 5);
+
+  return (
+    <Box>
+      <Box
+        height={BLOCK_SIZES.TWO_TWELFTHS}
+        padding={[4, 0, 4, 0]}
+        display={DISPLAY.FLEX}
+        alignItems={ALIGN_ITEMS.CENTER}
+        flexDirection={FLEX_DIRECTION.ROW}
+        className="add-network__header"
+      >
+        <img
+          src="./images/caret-left-black.svg"
+          alt={t('back')}
+          onClick={() => onBackClick}
+          className="add-network__header__back-icon"
+        />
+        <Typography variant={TYPOGRAPHY.H3} color={COLORS.BLACK}>
+          {t('addNetwork')}
+        </Typography>
+      </Box>
+      <Box
+        height={BLOCK_SIZES.FOUR_FIFTHS}
+        width={BLOCK_SIZES.TEN_TWELFTHS}
+        margin={[0, 6, 0, 6]}
+      >
+        <Typography
+          variant={TYPOGRAPHY.H6}
+          color={COLORS.UI4}
+          margin={[4, 0, 0, 0]}
+        >
+          {t('addFromAListOfPopularNetworks')}
+        </Typography>
+        <Typography
+          variant={TYPOGRAPHY.H7}
+          color={COLORS.UI3}
+          margin={[4, 0, 3, 0]}
+        >
+          {t('customNetworks')}
+        </Typography>
+        {nets.map((item, index) => (
+          <Box
+            key={index}
+            display={DISPLAY.FLEX}
+            alignItems={ALIGN_ITEMS.CENTER}
+            marginBottom={6}
+          >
+            <img
+              className="add-network__token-image"
+              src={item?.rpcPrefs?.imageUrl}
+              alt={t('logo', [item.ticker])}
+            />
+            <Typography variant={TYPOGRAPHY.H7} color={COLORS.BLACK}>
+              {item.ticker}
+            </Typography>
+            <img
+              className="add-network__add-icon"
+              src="./images/times.svg"
+              alt={`${t('add')} ${item.ticker}`}
+              onClick={() => onAddNetworkClick}
```

Comment:
[nit]
```suggestion
              onClick={onAddNetworkClick}
```

Reply:
Fixed in: 7880226d2cbb1bdcffb7d9bbc1c996f7ec46dbac